<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceTrainNumSetPage</class>
 <widget class="QWidget" name="MaintainceTrainNumSetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QPushButton
{
	border-radius:8px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:0.4,stop:0 rgba(150,215,230,255),stop:0.4 rgba(100,170,200,255)
spread:pad,x1:0,y1:0.4,x2:0,y2:1,stop:0.4 rgba(52,136,159,255),stop:1 rgba(94,188,216,255));
color:rgb(23,176,176);
font:28px,&quot;微软雅黑&quot;;
border: 1px solid transparent;
}
QLabel
{
	color:white;
	font:28px &quot;微软雅黑&quot;
}</string>
  </property>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>907</x>
     <y>530</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
		border-radius:8px;
background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>520</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>原车号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>新车号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentNumlbl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SettingNumlbl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid white;
background-color:blue;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>270</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
 color:white;font:18px,&quot;微软雅黑&quot;;border-top:2px solid  white;border-left: 2px solid white;border-bottom:2px solid transparent;border-right: 2px solid transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(150,228,250,255),stop:0.7 rgba(20,108,140,255) ,stop:1 rgba(100,238,240,255));
}</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>returnBtn</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>CurrentNumlbl</zorder>
  <zorder>SettingNumlbl</zorder>
  <zorder>ConfirmBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
